.circleci/config.yml

```diff
@@ -62,13 +60,18 @@ commands:
       - run:
           name: 'Install Dependencies'
           command: ./.circleci/install_dependencies.sh
+  setup_vm_browsers:
+    steps:
+      - browser-tools/install-chrome:
+          replace-existing: true
 
 jobs:
   'Checks':
     executor:
       name: amphtml-large-executor
     steps:
       - setup_vm
+      - setup_vm_browsers
```

Comment:
Might be able to do away with browser install for the `Checks` job. Doesn't look like anything needs it: https://app.circleci.com/pipelines/github/ampproject/amphtml/836/workflows/8e1dee6b-ef57-4a42-8b57-29222273d59b/jobs/7289

Reply:
Good feedback, I'll add this.